packages/core/src/components/DropdownNew/hooks/useDropdownFiltering.ts

```diff
@@ -1,39 +1,19 @@
 import { useState, useEffect } from "react";
 import { normalizeOptions } from "../utils/dropdownUtils";
-import { BaseListItemProps } from "../../BaseListItem";
+import { BaseListItemData } from "../../BaseListItem";
 import { ListGroup } from "../../BaseList";
 import { DropdownGroupOption } from "../Dropdown.types";
 
-function useDropdownFiltering<T extends BaseListItemProps>(options: DropdownGroupOption<T>) {
+function useDropdownFiltering<T extends BaseListItemData>(options: DropdownGroupOption<T>) {
```

Comment:
same for the extends

Reply:
I believe `BaseListItemData` should get the `T` here?
once `BaseListItemData` won't have default you would be forced to - which is great